pom.xml

```diff
@@ -77,9 +83,9 @@
 	
 	<dependencies>
```

Comment:
In the end run there probably will be a few that are global and some that are specific. I do believe that the process should start with nothing in the global and then as things are shared by more than two modules they are pulled into the global